app/invoices/invoice_helper.rb

```diff
@@ -274,7 +274,7 @@ def payment_page_for_initial_employer
     end
 
     @pdf.move_down 72
-    @pdf.text_box "Please review the billing summary. This is a consolidated bill for all your benefits through the Massachusetts #{site_short_name}. Please pay the Total Amount Due.", :at => [address_x, @pdf.cursor]
+    @pdf.text_box "Please review the billing summary. This is a consolidated bill for all your benefits through the Maine #{site_short_name}. Please pay the Total Amount Due.", :at => [address_x, @pdf.cursor]
```

Comment:
This should be changed in `disable_shop_base`- you can get the state name through the Contact center or elsewheree